### PR TITLE
feat(pipeline): add org annotation for pipeline job

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
add org annotation for pipeline job

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJhc3NpZ25lZSI6WyIxMDAxMjA1Il19&id=306028&iterationID=-1&pId=0&type=TASK)

#### Specified Reviewers:

/assign @sfwn

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  add org annotation for pipeline job            |
| 🇨🇳 中文    |   添加组织的元信息到pipeline job           |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chengjoey** [comment]
normal job:
image

spark job:
image

flink job: 
image

File: apistructs/job.go [hunk 1]
```diff
@@ -115,6 +115,26 @@ type JobFromUser struct {
 	ContainerInstanceProvider *ContainerInstanceProvider `json:"containerInstanceProvider,omitempty"` // ContainerInstanceProvider pointer type if not hitted of specified, it will be nil
 }
 
+func (job *JobFromUser) GetOrgName() string {
+	if job.Labels != nil {
+		orgName, ok := job.Labels["DICE_ORG_NAME"]
```
Comment (sfwn) [inline apistructs/job.go:120]: use DICE_ORG_NAME & DICE_ORG_ID in apistructs.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: apistructs/job.go (hunk 1 above)
Comment (chengjoey) [reply apistructs/job.go]: adjusted, and use apistructs define in prepare
image

**Review by sfwn — APPROVED** [review]

**Pull request merged**